config/locales/devise.en.yml

```diff
@@ -0,0 +1,65 @@
+# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
```

Comment:
Language files for internationalization, not sure if we'll eventually need this or not, but DWill said it was fine to keep